lib/shared/cache.js

```diff
@@ -1,29 +1,49 @@
 const fs = require('fs');
 const path = require('path');
 const chalk = require('chalk');
+const esbuild = require('esbuild');
+const crypto = require('crypto');
 const { logSlugMsg, logSlugError, runScript, prettyDuration, serializeErrorForClient } = require('./utils');
 const cacheCommand = path.join(__dirname, '../../bin/cache');
-const matchCacheFilename = /(?:^|\/)\.discoveryjs\.(.*?)\.(\d+)\.cache$/;
+const matchCacheFilename = /(?:^|\/)\.discoveryjs\.(.*?)\.(\d+)\.?([a-f0-9]+)?\.cache$/;
```

Comment:
```suggestion
const matchCacheFilename = /(?:^|\/)\.discoveryjs\.(.*?)\.(\d+)(\.[a-f0-9]+)?\.cache$/;
```

Reply:
fixed

Reply:
wait, no. in this case hash group will be `.aaaaaaa` and `.hash` and it will not be replaced